tools/gn

```diff
@@ -1188,7 +1158,7 @@ def parse_args(args):
 
   parser.add_argument(
       '--trace-gn',
-      default=False,
+      default=True,
```

Comment:
We can use this to debug the `gn` command being slow, and there's no real reason to have it off.